src/index.ts

```diff
@@ -520,6 +539,17 @@ export function create(rawOptions: CreateOptions = {}): Service {
     6059, // "'rootDir' is expected to contain all source files."
     18002, // "The 'files' list in config file is empty."
     18003, // "No inputs were found in config file."
+    ...(options.experimentalReplAwait
+      ? [
+          1103, // A 'for-await-of' statement is only allowed within an async function or async generator
+          1308, // 'await' expression is only allowed within an async function
+          1375, // 'export {}' requirement for top level 'await'
+          1378, // module & target requirement for top level 'await'
+          1431, // 'export {}' requirement for top level 'for await'
+          1432, // module & target requirement for top level 'for await'
+          2304, // Cannot find name 'await'
+        ]
+      : []),
```

Comment:
Will these diagnostics be ignored for files outside of the REPL, for example if someone requires a ts file from the REPL?  If the user enables `experimentalReplAwait` via their tsconfig, but they run a script like `ts-node ./foo.ts` will these diagnostics be ignored?  What will this look like if node enables REPL await by default and we do the same?

I want to understand what breakages we may have.

Reply:
Yes, since we added that option to tsconfig it will ignore these codes for entrypoint as well. We can either:
- remove option from tsconfig
- conditionally exclude these codes based on `executeEntrypoint`

Reply:
I've opted for the 2nd option, as the first option wouldn't prevent the issue from happening if using `ts-node --experimental-repl-await ./foo.ts` directly.